documentation/storageclass.md

```diff
@@ -15,47 +15,32 @@ vSphere is one of the provisioners and it allows following parameters:
 
 * **storagePolicyName** is an optional field which is the name of the SPBM policy to be applied. The newly created persistent volume will have the SPBM policy configured with it.
 
-* **VSAN Storage Capability Parameters** (cacheReservation, diskStripes, forceProvisioning, hostFailuresToTolerate, iopsLimit and objectSpaceReservation) are supported by vSphere provisioner for vSAN storage. The persistent volume created with these parameters will have these vSAN storage capabilities configured with it. For more detail about these parameters go to [Storage Policy Management section](/vsphere-storage-for-kubernetes/documentation/policy-based-mgmt.html).
+* **VSAN Storage Capability Parameters** (cacheReservation, diskStripes, forceProvisioning, hostFailuresToTolerate, iopsLimit and objectSpaceReservation) are supported by vSphere provisioner for vSAN storage. The persistent volume created with these parameters will have these vSAN storage capabilities configured with it. For more detail about these parameters go to [Storage Policy Management section](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/policy-based-mgmt.html).
 
 **Note:**
 
 All the example yamls can be found [here](https://github.com/Kubernetes/Kubernetes/tree/master/examples/volumes/vsphere) unless otherwise specified. Please download these examples.
 
 Let us look at an example of how to use StorageClass for dynamic provisioning.
 
-**Note:** Here you don't need to create vmdk it is created dynamically.
+**Note:** With StorageClass, VMDK need not be created manually for Persistent Volume. VMDK will be created dynamically by `vsphere-volume` provisioner.
 
 **Create Storage Class**
 
 ```
-#vpshere-volume-sc-fast.yaml
-
-kind: StorageClass
+$ cat vpshere-volume-sc-fast.yaml
 apiVersion: storage.k8s.io/v1
-metadata:
-  name: fast
-provisioner: kubernetes.io/vsphere-volume
-parameters:
-    diskformat: zeroedthick
-    fstype:     ext3
-```
-
-You can also specify the datastore in the Storageclass as shown below YAML for dynamic provisioning. The volume will be created on the datastore specified in the storage class. This field is optional. If not specified as shown in above YAML, the volume will be created on the datastore specified in the vsphere config file used to initialize the vSphere Cloud Provider.
-
-```
-#vsphere-volume-sc-slow.yaml
-
 kind: StorageClass
-apiVersion: storage.k8s.io/v1
-metadata:
+metadata: 
   name: fast
 provisioner: kubernetes.io/vsphere-volume
-parameters:
-    diskformat: zeroedthick
-    datastore: VSANDatastore
+parameters: 
+  datastore: VSANDatastore
+  diskformat: thin
+  fstype: ext3
 ```
 
-If datastore is member of DataStore Cluster or within some sub folder, the datastore folder path needs to be provided in the datastore parameter as below.
+If datastore is the member of DataStore Cluster or within some subfolder, the datastore folder path needs to be provided in the datastore parameter as below.
```

Comment:
Nit:
the member -> a member
DataStore -> Datastore?

Reply:
Fixed.